src/hasSubObject.test.ts

```diff
@@ -1,4 +1,4 @@
-import { hasSubObject } from "./hasSubObject";
+import { type HasSubObjectBrand, hasSubObject } from "./hasSubObject";
```

Comment:
don't import the type, we talked about using `toMatchTypeOf` instead of `toEqualTypeOf` to bypass the branded issue.
```suggestion
import { hasSubObject } from "./hasSubObject";
```

Reply:
Done